heartbeat/docker

```diff
@@ -236,7 +236,25 @@ monitor_cmd_exec()
 
 container_exists()
 {
-	docker inspect --format {{.State.Running}} $CONTAINER | egrep '(true|false)' >/dev/null 2>&1
+	local err
+
+	err=$(docker inspect --type=container $CONTAINER 2>&1 >/dev/null)
+
+	if [ $? -ne $OCF_SUCCESS ]; then
+		case $err in
+			*"No such container"*)
+				# Container does not exist
```

Comment:
I think we should add e.g. `ocf_exit_reason "Container doesnt exist"` or similar here, or just skip the entire case if the output from `docker inspect` is clear enough to output directly as in `*)`.

Reply:
> I think we should add e.g.` ocf_exit_reason "Container doesnt exist"` or similar here

Is that still relevant since we're not exiting if the container doesn't exist?

"No such container" => Return 1 from the function
Any other error => Exit with code 1 from the whole script

Reply:
It seems like you're right. Maybe just `return 1` instead of the error-code and update to something along the lines of "return instead of exit when container doesnt exist", as this wasnt easy to think was a mistake.

Reply:
Sorry I dropped the ball on this one till now. I initially had `return 1` and changed it to `OCF_ERR_GENERIC` in response to feedback. I think there was some misunderstanding between us at the time on what it was being used for. I'll push an updated version in a minute.

Reply:
Done.

Reply:
Botched commit history now... will try to fix.